### PR TITLE
Improved icons

### PR DESCRIPTION
**Pull request by evoactivity**
- center icons
- adjust sizing
- use true cut out for versions with text
- use colour for text that works on both light and dark

image

image

### CODE REVIEW

**Comment by lifeart** [comment]
@evoactivity only manually :) I mean, you need to change icons links in `package.json`

**Comment by evoactivity** [comment]
How does this look?

image
image

**Comment by evoactivity** [comment]
Slome more options

image


image

Another option would be light/dark variants

image

image

image
image

**Comment by evoactivity** [comment]
@kaermorchen how do you feel about the variants in the post above?

**Comment by kaermorchen** [comment]
The concept in whole is cool, but not for small icons.

- Red background and yellow `JS` is ok, but other variants have too low contrast.
- `JS` and `TS` labels are too small.

**Comment by lifeart** [comment]
Could we in addition explore this case:

image

(5 of 9 devs like it)

**Comment by kaermorchen** [comment]
> Could we in addition explore this case?

@lifeart look at #38

**Comment by evoactivity** [comment]
I don't agree about the label size, they match the vscode icons that have labels

image


I've made several variations for people to discuss

No text all 
image


Current version
image

High Contrast Version
image

I do not like any of the text only versions

No logo lowercase
image

No logo lowercase G
image

No logo uppercase
image

**Comment by evoactivity** [comment]
Making this a draft as if we agree to use my variant there are some tweaks I'll make before the PR is ready to merge.

**Comment by evoactivity** [comment]
I've increased the size of the text on these as much as I feel I can whilst still having the glimmer logo be recognisable

|new|old|
|---|---|
|image|image|

File: package.json [hunk 1]
```diff
@@ -44,7 +44,6 @@
     "vs:publish:ci": "vsce publish --yarn --pat $VSCODE_STORE_TOKEN",
     "ov:publish:ci": "ovsx publish --yarn --pat $OPEN_VSX_STORE_TOKEN",
     "node:package": "webpack --mode production",
-    "web:package": "webpack --mode production",
```
Comment (lifeart) [inline package.json:47]: @evoactivity I think we still need it :)

Comment (evoactivity) [reply package.json:47]: this was removed by mistake, I'll put it back

**Comment by lifeart** [comment]
@evoactivity thank you for contribution!

**Pull request merged**